modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -626,63 +754,82 @@ FeatureFloodCount::updateFieldInfo()
     _total_volume_intersecting_boundary.resize(_single_map_mode || _condense_map_info ? 1 : _maps_size);
   }
 
-  // Vector for indirect sort
-  std::vector<size_t> index_vector;
+//  decltype(FeatureFloodCount::FeatureData::_var_idx) old_var_index = 0;
 
-  unsigned int feature_number = 0;
-  for (auto map_num = decltype(_maps_size)(0); map_num < _maps_size; ++map_num)
+  /**
+   * To produce the proper global numbering, we need to loop over the local to
+   * global map and
```

Comment:
It sure as hell is... This is what happens when you are watching the Olympics while coding.